Source/DataSources/KmlDataSource.js

```diff
@@ -675,6 +720,18 @@ define([
                 }
                 polyline.width = queryNumericValue(node, 'width', namespaces.kml);
                 polyline.material = queryColorValue(node, 'color', namespaces.kml);
+                if (defined(queryColorValue(node, 'outerColor', namespaces.gx))) {
+                    console.log('KML - gx:outerColor is not supported in a LineStyle');
+                }
+                if (defined(queryNumericValue(node, 'outerWidth', namespaces.gx))) {
+                    console.log('KML - gx:outerWidth is not supported in a LineStyle');
+                }
+                if (defined(queryNumericValue(node, 'physicalWidth', namespaces.gx))) {
```

Comment:
Definitely not for this PR, but at some point we should experiment with corridor for this.